src/main/java/com/hazelcast/azure/AzureClientHelper.java

```diff
@@ -31,6 +37,8 @@
  */
 public final class AzureClientHelper {
```

Comment:
General question about this class (and our over-arching goal): How would I go about using this/deploying BbS in a non-default Azure environment?

Reply:
The current implementation only supports the default Azure environment.  The other environments are `AZURE_CHINA`, `AZURE_US_GOVERNMENT` and `AZURE_GERMANY`. If we want to support the other environment we would add a new property and switch based on it

Reply:
Yep, but is that a "regression" (lost feature) as compared to the previous state of the plugin? At least Germans are often quite keen to deploy in German data-centers for privacy and data-protection (regulatory) reasons.